arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/raise/NullableSpec.kt

```diff
@@ -69,14 +69,6 @@ class NullableSpec {
     } shouldBe "1"
   }
 
-  @Test fun bindingEitherInNullable() = runTest {
```

Comment:
I felt justified in removing these tests because the usage of `bind()` in this way is rather pointless because `Either#merge` does the same thing, and doesn't require a `Raise` at all. A `bind` method for `Either<Nothing, A>` can be added inside `SingletonRaise`, but again, it wouldn't be using anything at all related to `raise()`, since it would simply call `Either.merge()`